services/slam/builtin/cartographer_int_test.go

```diff
@@ -121,7 +122,7 @@ func TestCartographerIntegration(t *testing.T) {
 
 	// Remove maps so that testing in offline mode will run in mapping mode,
 	// as opposed to updating mode.
-	test.That(t, os.RemoveAll(name+"/map/*"), test.ShouldBeNil)
+	test.That(t, os.RemoveAll(name+"/map"), test.ShouldBeNil)
```

Comment:
`os.RemoveAll` removes the directory as well as files inside. Testing locally showed that the initial setup was looking for directories within map, rather than the map directory. Unsure on how tests were passing before this change